src/docs/app-monitoring/Sysdig-Monitoring-Guidline-for-shared-services-and-apps.md

```diff
@@ -0,0 +1,322 @@
+---
+title: SRE Guideline for Platform Shared Services
```

Comment:
In the PR it the file .md still reads Sysdig-Monitoring-**Guidline**-for-shared-services-and-apps please make sure to update it correctly, I believe @ShellyXueHan made a suggestion 

Slug of the website should also be all lower case: sysdig-monitor-setup